### PR TITLE
Petersburg

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
new record

### CODE REVIEW

File: PRS12001-13000/PRS12387PapadopuloKeramevs.xml [hunk 1]
```diff
@@ -59,19 +59,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <listPerson>
                 <person sex="1" sameAs="wd:Q4344608">
-                    <persName xml:lang="gez" xml:id="n1">Afanasij Ivanovič Papadopulo-Keramevs</persName>                  
-                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <persName xml:lang="rus" xml:id="n1">Afanasij Ivanovič Papadopulo-Keramevs</persName>
+                    <persName xml:lang="rus" type="normalized" corresp="#n1"/>
```
Comment (DariaElagina) [inline PRS12001-13000/PRS12387PapadopuloKeramevs.xml]: One tag is empty. Should it rather be:
```
<persName xml:lang="rus" xml:id="n1">Афанасий Иванович Пападопуло-Керамевс</persName>
                    <persName xml:lang="rus" type="normalized" corresp="#n1">Afanasij Ivanovič Papadopulo-Keramevs</persName>

```

File: new/PRS12867Ginstburg.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS12867Ginstburg" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Gintsburg</title>
```
Comment (DariaElagina) [inline new/PRS12867Ginstburg.xml]: Gincburg?
http://kodeks.uni-bamberg.de/AKSL/Schrift/Transliteration.htm

Comment (DariaElagina) [reply new/PRS12867Ginstburg.xml]: Or Günzburg?

File: new/PRS12867Ginstburg.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1" sameAs="wd:Q5234499">
+                  <persName xml:id="n1">
+                       <forename>David</forename>
+                       <forename>Horatsievič</forename>
+                       <surname>Gintsburg</surname>
```
Comment (DariaElagina) [inline new/PRS12867Ginstburg.xml]: Goracievič Gincburg xml:lang=rus, type=normalized

**Comment by DenisNosnitsin1970** [comment]
corrected

**Comment by thea-m** [comment]
@DariaElagina can this be merged?

**Comment by DenisNosnitsin1970** [comment]
This can be merged.

**Comment by PietroLiuzzo** [comment]
It cannot because of conflict on these files (same once again...) I will have a look at it now.

**Comment by DenisNosnitsin1970** [comment]
I remember vaguely that there was smth with these files in the past. I cannot resolve the conflics - the button seems to have been deactivated, i cannot press it.

**Comment by PietroLiuzzo** [comment]
As said, I will have a look at this. please do not commit further changes, I will fix this and proceed to merge my self as soon as possible. thanks

**Comment by PietroLiuzzo** [comment]
I am closing (WITHOUT MERGING) this PR, and I will reiterate the changes in a fresh branch which I will then merge directly. the conflicts where too deep in for me to be able to extract a cause, so I will just reproduce the changes.

**Pull request closed without merging**